src/main/python/powerFactory2json/pf2json.py

```diff
@@ -2,7 +2,11 @@
 import json
 import inspect
 import os
-from pf2jsonUtils import excluded_fields, elements4export, nested_elements4export, reserved_keywords
+import pf2jsonUtils
+import imp
```

Comment:
I think you are right. But that was the only way to make sure that the pfjson2utils module is updated in the export script after changing stuff (adding/changing parameters). Since this will happen quite regularly until this thing is done I think we need to keep that.

Reply:
Yes, otherwise you would have the need to reload the interpreter. If we do so, please add a FIXME-comment and an issue, so that we remember to remove it later.

Reply:
Added Issue #48